app/services/workers/calculate-workload-points.js

```diff
@@ -16,11 +16,18 @@ module.exports.execute = function (task) {
   var id = task.additionalData.workloadBatch.startingId
   var batchSize = task.additionalData.workloadBatch.batchSize
   var reportId = task.workloadReportId
+  var maxId = id + batchSize - 1
+
+  if (batchSize === 1) {
```

Comment:
I think if you set maxId = id above and changed this to batchSize > 1 you could possibly do away with the else which is only being used for logging a variant of the message.

E.g.
```
maxId = id
message = 'single workload message'
if (batchSize > 1) {
 maxId = maxId + batchSize - 1
 append additional stuff to message
}
log(message)